auth/service/proto/auth.proto

```diff
@@ -35,25 +35,22 @@ message Token {
 message Account {
 	string id = 1;
 	string type = 2;
-	repeated string roles = 3;
 	map<string, string> metadata = 4;
-	string namespace = 5;
-	string provider = 6;
+	repeated string scopes = 5;
```

Comment:
Be careful, this is a breaking change

Reply:
Well, this whole file is really

Reply:
Yes, since we're still using this only internally it's okay to do the breaking change as i'll bump all the dependancies when we roll this out.